decidim-core/spec/controllers/omniauth_registrations_controller_spec.rb

```diff
@@ -31,6 +31,49 @@ module Decidim
         }
       end
 
+      context "when the organization does have a valid authorization handler" do
+        let!(:user) { create(:user, :confirmed, organization:, sign_in_count: 1) }
+
+        context "when there are authorization handlers" do
+          before do
+            allow(user.organization).to receive(:available_authorizations)
+              .and_return(["dummy_authorization_handler"])
+            post :create
+          end
+
+          it { expect(controller).to redirect_to("/authorizations") }
```

Comment:
Let me know if you think this should be handled in another PR/issue @alecslupu, but I was expecting that this URL was actually http://localhost:3000/authorizations/first_login (first_login_authorizations_path), as that's a friendlier page for a first login without authorization. Maybe it's (another) bug? 

https://github.com/decidim/decidim/blob/35b03977627d4d70e8ea11250d2b39f434829104/decidim-core/app/controllers/decidim/devise/sessions_controller.rb#L44 

Also I've checked out what happens without omniauth and [it doesn't look that we had this bug](https://github.com/decidim/decidim/blob/35b03977627d4d70e8ea11250d2b39f434829104/decidim-core/app/controllers/decidim/devise/sessions_controller.rb#L59C1-L59C107)

Reply:
To be honest, it seems that we have different behaviour on sessions and registrationns controller. I am not 100% sure is this is the same bug or not. What i can do, in the scope of this PR, is to extract a concern that would be used in sessions and registration module. So that we have a consistent behavior.

Reply:
I have extracted the method required methods to a single place. You can see more in d49cd82a91541a5ce3c3a0e3fa26e60aeac9fb43

Reply:
I agree, this mismatch of redirections smells like the typical bug introduced by copying and pasting these methods. By making this concern them, I think we fix this issue and also the one that started it all (using `ecidim::Verifications.workflows.any?` instead of `current_organization.available_authorizations.any?`